yeonjae22/FBMember/app/src/main/java/user/manage/fbmember/MypageActivity.kt

```diff
@@ -0,0 +1,46 @@
+package user.manage.fbmember
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseUser
+import kotlinx.android.synthetic.main.mypage.*
+
+
+class MypageActivity : AppCompatActivity(){
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.mypage)
```

Comment:
```suggestion
        setContentView(R.layout.activity_mypage)
```
이렇게 activity_pagename 형식으로 이름짓는것이 android layout convention에 있습니다.

Reply:
변경했습니다!